GenumKit/Parsers/StringsFileParser.swift

```diff
@@ -17,12 +31,41 @@ public final class StringsFileParser {
 
   // Localizable.strings files are generally UTF16, not UTF8!
   public func parseStringsFile(path: String) throws {
+    guard let data = NSData(contentsOfFile: path) else {
+      throw StringsFileParserError.FailureOnLoading(path: path)
+    }
+
+    let plist = try NSPropertyListSerialization.propertyListWithData(data, options: NSPropertyListReadOptions.Immutable, format: nil)
+
+    guard let dict = plist as? Dictionary<String, String> else {
+      throw StringsFileParserError.InvalidFormat
+    }
+
     var encoding: NSStringEncoding = NSUTF16StringEncoding
     let fileContent = try NSString(contentsOfFile: path, usedEncoding: &encoding)
+
+    func extractKey(line: String) -> String? {
```

Comment:
Not sure why you'd need that.
Now that we parse the `.strings` file as being an `NSDictionary`, we don't deal with _lines_ in the text files, but already have key/value pairs, so you already have the key. No need to get the lines from the textual format anymore nor extract their keys!